vtkext/private/module/vtkF3DGenericImporter.cxx

```diff
@@ -155,6 +154,8 @@ void vtkF3DGenericImporter::ImportActors(vtkRenderer* ren)
   this->Pimpl->GeometryBoundingBox.Reset();
   bool hasGeometry = false;
 
+  InitializeVolumeMapper();
```

Comment:
instead of initializing here you can drop the method and put the code just before using the mapper, line 217

Reply:
moved the initialization